src/idl/src/parser.y

```diff
@@ -837,6 +842,27 @@ enumerator:
       }
   ;
 
+bitmask_dcl: bitmask_def { $$ = $1; } ;
+
+bitmask_def:
+    "bitmask" identifier '{' bit_values '}'
```

Comment:
I don't see a token added for `"bitmask"`? I'm a bit surprised it works. Does it show up in the token table?

Reply:
The token was already in [parser.y](https://github.com/eclipse-cyclonedds/cyclonedds/blob/5fa204dd61620b24c394ef88d9040944ecbbb470/src/idl/src/parser.y#L270)